package.json

```diff
@@ -77,13 +77,14 @@
     "ember-cli-sass": "^10.0.1",
     "ember-cli-terser": "^4.0.0",
     "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-code-example-component": "^0.4.0",
-    "ember-code-snippet": "^2.4.0",
+    "ember-code-example-component": "github:cibernox/ember-code-example-component#24be4ddb0cf998a0dd77faff779ea9cd1da88859",
```

Comment:
@cibernox ToDo: change to explicit version when you release the latest changes in `ember-code-example-component` 🙏